### PR TITLE
Obsolete `ContinueOnCapturedContext`

### PR DESCRIPTION
**Pull request by viceroypenguin**
Part of #4475

### CODE REVIEW

**Review by sdanyliv — APPROVED** [review]

File: Source/LinqToDB/Common/Configuration.cs [hunk 1]
```diff
@@ -71,6 +71,7 @@ public static class Configuration
 		/// Defines value to pass to <see cref="Task.ConfigureAwait(bool)"/> method for all linq2db internal await operations.
 		/// Default value: <c>false</c>.
 		/// </summary>
+		[Obsolete("Value is ignored in v6. Will be removed in v7.")]
```
Comment (MaceWindu) [inline Source/LinqToDB/Common/Configuration.cs:74]: I would prefer if we just remove it now

Comment (MaceWindu) [reply Source/LinqToDB/Common/Configuration.cs:74]: Actually, doesn't matter

**Review by MaceWindu — APPROVED** [review]
ok, just revert api baselines change

**Review by MaceWindu — DISMISSED** [review]

File: Source/LinqToDB/PublicAPI/PublicAPI.Shipped.txt [hunk 2]
```diff
@@ -8609,7 +8609,7 @@ static LinqToDB.AsyncExtensions.ToListAsync<TSource>(this System.Linq.IQueryable
 static LinqToDB.Common.Compilation.CompileExpression(this System.Linq.Expressions.LambdaExpression! expression) -> System.Delegate!
 static LinqToDB.Common.Compilation.CompileExpression<TDelegate>(this System.Linq.Expressions.Expression<TDelegate!>! expression) -> TDelegate!
 static LinqToDB.Common.Compilation.SetExpressionCompiler(System.Func<System.Linq.Expressions.LambdaExpression!, System.Delegate?>? compiler) -> void
-static LinqToDB.Common.Configuration.ContinueOnCapturedContext -> bool
+static LinqToDB.false -> bool
```
[the remark this answers is not in the document]
Comment (viceroypenguin) [reply Source/LinqToDB/PublicAPI/PublicAPI.Shipped.txt]: 😄

File: Source/LinqToDB/Common/Configuration.cs (hunk 1 above)
Comment (viceroypenguin) [reply Source/LinqToDB/Common/Configuration.cs:74]: Actually, i'm going to change the xmldoc for this too

**Review by MaceWindu — APPROVED** [review]

**Review by MaceWindu — APPROVED** [review]

**Pull request merged**